python-spec/src/somacore/data.py

```diff
@@ -195,8 +195,17 @@ def create(
         :param uri: The URI where the array will be created.
         :param type: The Arrow type to store in the array.
             If the type is unsupported, an error will be raised.
-        :param shape: The length of each dimension as a sequence,
-            e.g. ``(100, 10)``. All lengths must be in the postive int64 range.
+        :param shape: The maximum capacity of each dimension, including room
+            for any intended future appends, as a sequence.  E.g. ``(100, 10)``.
+            All lengths must be in the postive int64 range, or ``None``.
+            It's necessary to say ``shape=(None, None)`` or ``shape=(None, None, None)``
+            rather than more simply ``shape=None`` since the first two are how one
```

Comment:
Nit: suggested edit for clarity: 

> It's necessary to say ``shape=(None, None)`` or ``shape=(None, None, None)``, as the sequence length determines the number of dimensions N, in the N-dimensional array.

Reply:
Awesome improvement, thanks @bkmartinjr !